label_studio/data_manager/managers.py

```diff
@@ -195,7 +194,13 @@ def add_result_filter(field_name, _filter, filter_expressions, project):
     elif _filter.operator == Operator.NOT_CONTAINS:
         filter_expressions.append(~Q(subquery))
         return 'continue'
-
+    elif _filter.operator == Operator.EMPTY and field_name == 'annotations_results':
+        if _filter.value == 'true':
```

Comment:
Let's edit this a little: 
```  
cast_value(_filter)  # it works inplace
if _filter.value: 
  ...
```

Reply:
yes this is cleaner. Unfortunately the UI is sending us the "annotation_results is empty" filter typed as a String so `cast_value` won't cast it into a Boolean
```
{
  "filter": "filter:tasks:annotations_results",
  "operator": "empty",
  "value": "true",
  "type": "String"
}
```